_data/committers.yml

```diff
@@ -203,10 +235,18 @@
   role: Committer
   alias: liyafan
   affiliation: Alibaba
+- name: Matthew Topol
+  role: Committer
+  alias: zeroshade
+  affiliation: FactSet Research Systems, Inc.
 - name: Neville Dipale
```

Comment:
Ah, please also remove this duplicate entry for Neville. We may want to update the script to check for duplicates...

Reply:
Done.